# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/client/MultiTiledContentClient.cpp
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 gfx/layers/client/MultiTiledContentClient.cpp
--- a/gfx/layers/client/MultiTiledContentClient.cpp	Sun May 26 14:29:56 2019 +0000
+++ b/gfx/layers/client/MultiTiledContentClient.cpp	Sun May 26 14:30:14 2019 +0000
@@ -7,6 +7,7 @@
 #include "mozilla/layers/MultiTiledContentClient.h"
 
 #include "ClientTiledPaintedLayer.h"
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/layers/LayerMetricsWrapper.h"
 
 namespace mozilla {
@@ -264,7 +265,7 @@
       ctx = nullptr;
 
       // Edge padding allows us to avoid resampling artifacts
-      if (gfxPrefs::TileEdgePaddingEnabled() && mResolution == 1) {
+      if (StaticPrefs::TileEdgePaddingEnabled() && mResolution == 1) {
         drawTarget->PadEdges(newValidRegion.MovedBy(-mTilingOrigin));
       }
 